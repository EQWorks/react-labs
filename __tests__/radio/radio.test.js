import React from 'react'
import { render } from '@testing-library/react'

import Radio from '../../src/radio'

describe('render', () => {
  it('should render ok with no props passed', () => {
    const { getByTestId } = render(<Radio />)
    const radio = getByTestId('radio')
    // snapshot
    expect(radio).toMatchSnapshot()
  })
})
