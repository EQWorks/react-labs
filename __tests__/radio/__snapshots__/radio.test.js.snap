// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render ok with no props passed 1`] = `
<span
  aria-disabled="false"
  class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root makeStyles-root-1"
  data-testid="radio"
>
  <span
    class="MuiIconButton-label"
  >
    <input
      class="PrivateSwitchBase-input-7"
      type="radio"
      value=""
    />
    <span
      class="makeStyles-icon-2"
    />
  </span>
</span>
`;
